res/css/views/dialogs/_LocationViewDialog.scss

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_LocationViewDialog_wrapper .mx_Dialog {
+    padding: 0px;
+
+    /* Unset contain and position to allow the close button
+    to appear outside the dialog */
+    contain: unset;
+    position: unset;
+}
+
+.mx_LocationViewDialog {
+    /* subtract 0.5px to prevent single-pixel margin due to rounding */
+    width: calc(80vw - 0.5px);
+    height: calc(80vh - 0.5px);
```

Comment:
these seem redundant in my testing, they cause a 0.5/1px jump on a @2x hidpi display.

Reply:
Without it (i.e. with `width: 80vw`), if I slowly shrink my window, I sometimes get an extra pixel like this:
![image](https://user-images.githubusercontent.com/76812/148539109-5870090a-e420-4c14-9ca0-56462f91411b.png)
My understanding/guess of the cause is that the canvas is sized with effectively `floor(80vw)` pixels, but the div is rounding to the nearest pixel.  Hence why I subtracted 0.5.

Please can you explain the symptom you are seeing?